__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const app = require("../app");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+afterAll(() => {
+  db.end();
+});
+
+describe("/api/categories", () => {
+  describe("GET /api/categories", () => {
+    test("status: 200, should respond with all treasures", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((result) => {
+          expect(result.body.categories).toHaveLength(4);
+          result.body.categories.forEach((category) => {
+            expect(category).toEqual(
```

Comment:
have a look at lines 21 and 22, we don't need 2 expect statements here (just one for each category).

Reply:
I've tried getting rid of expect but it doesn't seem to work without it

Reply:
All good in that case